pkgs/development/python-modules/datasette/template-sql.nix

```diff
@@ -0,0 +1,25 @@
+{ buildPythonPackage, fetchFromGitHub, lib,
+  datasette, pytest, pytest-asyncio, sqlite-utils }:
+
+buildPythonPackage rec {
+  pname = "datasette-template-sql";
+  version = "1.0.2";
+
+  src = fetchFromGitHub {
+    owner = "simonw";
+    repo = pname;
+    rev = version;
+    sha256 = "1ag5f62y445jscxnklcfd84pjinkknmrpk1kmm2j121p7484hrsn";
+  };
+
+  propagatedBuildInputs = [ datasette ];
+
+  checkInputs = [ pytest pytest-asyncio sqlite-utils ];
```

Comment:
Looks like that we could use `pytestCheckHook`.

Please add `pythonImportsCheck` as a additional safe-guard.

Reply:
Done, thanks.